mlflow/recipes/steps/train.py

```diff
@@ -149,6 +152,7 @@ def _validate_and_apply_step_config(self):
                 error_code=INVALID_PARAMETER_VALUE,
             )
         self.positive_class = self.step_config.get("positive_class")
+        self.rebalancing_enabled = self.step_config.get("rebalancing_enabled", True)
```

Comment:
Prefer to use a verb, e.g. `use_balanced_training_data` or `rebalance_training_data`